tests/device_tests/test_msg_sd_protect.py

```diff
@@ -16,47 +16,36 @@
 
 import pytest
 
-from trezorlib import debuglink, device, messages as proto
+from trezorlib import device
 from trezorlib.exceptions import TrezorFailure
+from trezorlib.messages import SdProtectOperationType as Op
 
-from ..common import MNEMONIC12
+pytestmark = pytest.mark.sd_card
 
 
-@pytest.mark.skip_t1
-class TestMsgSdProtect:
-    @pytest.mark.setup_client(mnemonic=MNEMONIC12)
-    def test_sd_protect(self, client):
+def test_sd_protect_enable(client):
+    # Disabling SD protection should fail
+    with pytest.raises(TrezorFailure):
```

Comment:
can we assert status of SD card protection? or is that a secret information?

Reply:
I guess @andrewkozlik needs to answer that one.

Reply:
The way I see it is that this information falls into the same category as the pin_protection flag. So I don't think it needs to be kept secret.